src/lib.rs

```diff
@@ -4,4 +4,568 @@
 
 //! resolver is a library for resolving DNS records for cueball.
 #[allow(dead_code)] // TODO: Remove after initial dev
-pub mod resolver;
+
+use chrono::prelude::*;
```

Comment:
I'm going to use this as an opportunity to solicit your opinion on a convention I've adopted. I've been organizing my imports into three groups. The first is the `std` imports, the second is the non-`std` external imports and the third is any crate-local imports. [Here's](https://github.com/joyent/rust-cueball/blob/master/src/connection_pool.rs) an example. I find it a nice way to bring a bit of order to what can get to be an overwhelmingly long list of jumbled imports, but curious if you see any value in such a practice.

Reply:
I like it -- will update.

Reply:
I chatted with rm about this briefly on Friday an we might even further break this down and separate third party external imports from the joyent external imports.